src/main/java/net/minecraftforge/common/command/SelectorHandler.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Minecraft Forge
+ * Copyright (c) 2016.
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.minecraftforge.common.command;
+
+import java.util.List;
+
+import net.minecraft.command.CommandException;
+import net.minecraft.command.ICommandSender;
+import net.minecraft.entity.Entity;
+
+/**
+ * Handler for custom types of selectors registered with {@link SelectorHandlerManager}
+ */
+public abstract class SelectorHandler
```

Comment:
It's needed for the default implementation of `isSelector` (assuming you want backwards compatibility to Java 7 and lower). This way, you shouldn't have to worry about this method too much as long as you don't do anything strange.